packages/runtime/garbage-collector/src/utils.ts

```diff
@@ -136,6 +136,8 @@ export function unpackChildNodesGCDetails(gcDetails: IGarbageCollectionDetailsBa
  * Helper function that unpacks the used routes of children from a given node's used routes.
  * @param usedRoutes - The used routes of a node.
  * @returns A map of used routes of each children of the the given node.
+ *
+ * @deprecated Internal implementation detail and will no longer be exported in an upcoming release.
```

Comment:
@ChumpChief This is the last function in garbage-collector package. Is there a way to deprecate packages?

Reply:
There's a way to do it on npm but I'm not familiar and we haven't done so historically (but we probably should).

I usually just make a note in the README and package.json description, esp. in cases like this where we suspect no real customers.

Reply:
https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions

Reply:
Added deprecated description to package.json of "garbage-collector". Can you please take a look if it looks fine?

Reply:
Looks good - I would also update the text in the README.md too since this is what is actually displayed on the package page on npmjs.com